systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -2125,6 +2131,321 @@ void testKafkaOffsetsReplicationFactorHigherThanReplicas() {
         KafkaResource.kafkaClient().inNamespace(NAMESPACE).delete(kafka);
     }
 
+    @Test
+    void testDynamicConfiguration() {
```

Comment:
Do we have somewhere some nice human readable list of test cases we have here? It would be useful to see what we have and what tests might be missing. Also I wonder if this deserves its own test class.

Reply:
Basically the tests are doing this
1. deploy clean Kafka cluster
2. dynamically update some change
3. update some change, which causes RU (listeners, logging,...)
4. dynamically update some change

After each step, it is tested whether Kafka was rolled. In the case of dynamic changes, it is tested whether the change is done in Kafka (using `kafka-configs.sh` to check dyn. conf of brokers)

Reply:
But the list of the changes done in 2 and 3 is what is interesting

Reply:
For dynamically changeable configuration I am setting `unclean.leader.election.enable` property. There is test wich is changing plain listeners to tls, to external, external nodeports to external loadbalancer.